pages/staff.tsx

```diff
@@ -68,7 +68,9 @@ const Staff = () => {
             <Search search={searchValue} setSearch={setSearch} />
           </div>
         </Box>
-        <Table rows={jsonData || []} columns={columns} />
+        {/* <Table rows={jsonData || []} columns={columns} /> */}
```

Comment:
I added extra props to the Table component that are specific to the Books table, which causes other instances of the Table component to run into errors. I'm not sure how to resolve this at the moment.